### PR TITLE
Editorial: fix 'in parallel' usage

### LINKED ISSUE #126: "request a position" uses "current setting object" from parallel steps.
Parallel steps shouldn't be poking at global mutable state.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #126

***
<a href="https://pr-preview.s3.amazonaws.com/marcoscaceres/geolocation-api/pull/129.html" title="Last updated on Sep 22, 2022, 1:32 AM UTC (134c8c5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/129/eaa69f3...marcoscaceres:134c8c5.html" title="Last updated on Sep 22, 2022, 1:32 AM UTC (134c8c5)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Hiding whitespace makes this easier to review: 
https://github.com/w3c/geolocation-api/pull/129/files?diff=unified&w=1

File: index.html [hunk 1]
```diff
@@ -722,56 +724,61 @@ <h2>
...
+              <li>While |watchIDs| [=list/contains=] |watchId|:
+                <ol>
+                  <li>
+                    <span id="wait-for-change">Wait for a significant change of
+                    geographic position</span>. What constitutes a significant
+                    change of geographic position is left to the
+                    implementation. User agents MAY impose a rate limit on how
+                    frequently position changes are reported.
+                  </li>
+                  <li>If |document| is not [=Document/fully active=] or
```
Comment (reillyeon) [inline index.html:760]: Can we check this while "in parallel"? If not then we should do a larger rewrite (perhaps in a follow-up to reduce churn) to only go "in parallel" for specifically the requesting permission and acquiring a position steps.

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:760]: yeah, good point. We should probably just queue a global task here that can then check the state of the document sync.

**Review by reillyeon — APPROVED** [review]
This looks good as an incremental step.

**Comment by reillyeon** [comment]
Please resolve the merge conflicts and I will merge this change.

**Comment by marcoscaceres** [comment]
Fixed! thanks for the speedy review!

**Review by reillyeon — APPROVED** [review]

**Pull request merged**